fms/models/__init__.py

```diff
@@ -343,9 +410,24 @@ def get_model(
                 devices, _guess_num_layers(lazy_sd)
             )
 
-    # Create the model
+    if extra_args.get("unfuse_strategy", None):
+        _validate_unfuse_strategy(extra_args, rank)
+
+        # change source for "gptq + pre" (= unfused gptq ckpt into unfused model)
```

Comment:
For a future PR, add compatibility with the new `hf_configured` and `hf_pretrained` source options we have added for model config inference from HF.